core/domain/stats_services.py

```diff
@@ -571,10 +571,6 @@ def get_updated_exp_issues_models_for_new_exp_version(
         if 'state_names' in playthrough.issue_customization_args:
             state_names = (
                 playthrough.issue_customization_args['state_names']['value'])
-            # TODO(#15995): Currently, we are define all issue customization
```

Comment:
Every assert needs a justification.

Actually I think it would be better (for issue customization args) to have a union of the possible types for each type of issue -- the choices there are pretty limited. But if you want to go with asserts and justification, that's fine too.

Reply:
reasons added for every assert PTAL!